tgui/packages/tgui/interfaces/LobbyMenu.tsx

```diff
@@ -0,0 +1,608 @@
+              </Stack.Item>
+            )}
+            {!!predator_enabled && (
+              <Stack.Item>
+                <LobbyButton
+                  index={7 + (upp_enabled ? 1 : 0)}
+                  onClick={() => {
+                    setModal(
+                      <ModalConfirm>
+                        <Box>
+                          <Stack vertical>
+                            <Stack.Item>
+                              Are you sure want to attempt joining as a
+                              Predator?
+                            </Stack.Item>
+                          </Stack>
+                          <Stack justify="center">
+                            <Stack.Item>
+                              <Button onClick={() => act('late_join_pred')}>
+                                Confirm
+                              </Button>
+                            </Stack.Item>
+                          </Stack>
+                        </Box>
+                      </ModalConfirm>,
+                    );
+                  }}
+                >
+                  <Flex>
+                    <Flex.Item>
+                      <Box className="gnarp" inline />
+                    </Flex.Item>
+                    <Flex.Item>Join the Hunt</Flex.Item>
+                  </Flex>
+                </LobbyButton>
+              </Stack.Item>
+            )}
+            {!!fax_responder_enabled && (
+              <Stack.Item>
+                <LobbyButton
+                  index={7 + (upp_enabled ? 1 : 0) + (predator_enabled ? 1 : 0)}
+                  icon="fax"
+                  onClick={() => {
+                    setModal(
+                      <ModalConfirm>
+                        <Box>
+                          <Stack vertical>
+                            <Stack.Item>
+                              Are you sure want to attempt joining as a Fax
+                              Responder?
+                            </Stack.Item>
+                          </Stack>
+                          <Stack justify="center">
+                            <Stack.Item>
+                              <Button onClick={() => act('late_join_faxes')}>
+                                Confirm
+                              </Button>
+                            </Stack.Item>
+                          </Stack>
+                        </Box>
+                      </ModalConfirm>,
+                    );
+                  }}
+                >
+                  Respond to Faxes
+                </LobbyButton>
+              </Stack.Item>
+            )}
+          </>
+        )}
+      </Stack>
+    </Section>
+  );
+};
+
+const TimedDivider = () => {
+  const ref = useRef<HTMLDivElement>(null);
+
+  const context = useContext(LobbyContext);
+
+  const { themeDisable } = context;
+
+  useEffect(() => {
+    if (!themeDisable) {
+      setTimeout(() => {
+        ref.current!.style.display = 'block';
+      }, 1500);
+    }
+  });
```

Comment:
I assume you only want this running if themeDisable's value is changed. Otherwise, it will run every re-render.
```suggestion
  useEffect(() => {
    if (!themeDisable) {
      setTimeout(() => {
        ref.current!.style.display = 'block';
      }, 1500);
    }
  }, [themeDisable]);
```